src/cai_lab_to_nwb/zaki_2024/zaki_2024_notes.md

```diff
@@ -868,4 +868,8 @@ https://youtu.be/BKgh-XcZhIM?t=731
 
 
 More about synch can be found in the video to align with behavior on this timestamp
-https://youtu.be/BKgh-XcZhIM?t=1338
\ No newline at end of file
+https://youtu.be/BKgh-XcZhIM?t=1338
+
+
+## Figures to reproduce in the example notebook:
+1) ![img.png](assets/motion_freezing_across_session.png)
```

Comment:
Good idea,

Reply:
I will add other for the segmentation data on this PR #7